woocommerce/woocommerce-ios
# Updates to Zendesk 5.0 SDK and CocoaPods 1.9.1

Closes #1928 

# Description

Zendesk has changed the CocoaPod for their SDK from `ZendeskSDK` to `ZendeskSupportSDK`. This also changes some of the fundamental class names as well within the SDK. More [details here](https://developer.zendesk.com/embeddables/docs/ios_support_sdk/migration_guide).

This PR updates the Zendesk dependency in the Podfile and also updates to the latest CocoaPods (1.9.1).

# Testing 

1. Clean your build.
2. Build and run in a simulator.
3. Make sure the Zendesk help menu is working and you can send a test message through.
4. Bonus points for testing on a real device.

Update release notes:
